modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -118,6 +119,12 @@ def validate_type_and_services_known
         raise Common::Exceptions::InvalidFieldValue.new('services', unknown) unless unknown.empty?
       end
 
+      def validate_state_code
+        if params[:state] && !STATE_CODES.include?(params[:state])
```

Comment:
rails supports `exclude?` as a clearer version of `!include?`

Reply:
Oh, fantastic. Will do!